lighthouse-core/formatters/table.js

```diff
@@ -78,13 +78,18 @@ class Table extends Formatter {
 
     const rows = results.map(result => {
       const cols = headingKeys.map(key => {
+        let value = result[key];
+        if (typeof value === 'undefined') {
```

Comment:
not sure how many undefined vals we get, but will a bunch of '--' cells be too noisy?